packages/charting/src/components/DonutChart/examples/DonutChart.Basic.Example.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+import { customizable } from '../../../Utilities';
+import { DonutChart, IDonutChartProps } from '../../DonutChart';
+
+@customizable('DonutChartBasicExample', ['theme', 'styles'])
+export class DonutChartBasicExample extends React.Component<IDonutChartProps, {}> {
+  constructor(props: IDonutChartProps) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+    return <div>{this._basicExample()}</div>;
+  }
+
+  private _basicExample(): JSX.Element {
+    const points = [{ value: 50, label: 'A' }, { value: 25, label: 'B' }, { value: 25, label: 'C' }];
+    const colors = ['#FF5733', '#176213', '#193BBD'];
+    return <DonutChart data={points} chartLabel={'Pie Chart'} colors={colors} />;
```

Comment:
There's a TypeScript error on this line shown in CI. Just in case you don't know you can check for errors by running `npm run build` (or more specifically `npm run ts` and `npm run tslint`) inside of `packages/charting`.